dev.inv

```diff
@@ -0,0 +1,5 @@
+[anotherinstance18.04]
+3.109.53.224 ansible_user=ubuntu ansible_ssh_private_key_file='/var/lib/jenkins/testAnsible.pem'
```

Comment:
why to use pem file name as camelcase, is that we follow as name convention?

Reply:
Will make sure from next time that I will not use camel casing format